launcher/game/updater.rpy

```diff
@@ -178,6 +218,20 @@ screen updater:
 label update:
 
     python hide:
+        criteria = ""
+        criteria = interface.input(
+            _("Mod search phrases"),
+            _("""Please insert search phrases separated by commas.
+Spaces placed before and after search phrases will be included in them.
```

Comment:
Confused as to what these words from here to the end mean. I only see a small difference searching for `true` than `'true'` so just saying Type in the mod name you want to search for works better